src/Entity/ShipmentServiceOptions.php

```diff
@@ -91,9 +97,9 @@ public function __construct($response = null)
             }
             if (isset($response->DirectDeliveryOnlyIndicator)) {
                 $this->DirectDeliveryOnlyIndicator = $response->DirectDeliveryOnlyIndicator;
-            }
-            if (isset($response->DeliverToAddresseeOnlyIndicator)) {
-                $this->DeliverToAddresseeOnlyIndicator = $response->DeliverToAddresseeOnlyIndicator;
```

Comment:
Why is this one removed? As far as I know it's a valid option (I'm actually using it in a project).